src/aiy/_drivers/_status_ui.py

```diff
@@ -0,0 +1,76 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A status UI powered by the LED on the VoiceHat."""
+
+import logging
+import os.path
+
+import aiy.audio
+
+# Location of the LED status-ui service's FIFO file.
+_LED_FIFO = "/tmp/status-led"
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="[%(asctime)s] %(levelname)s:%(name)s:%(message)s"
+)
+logger = logging.getLogger('status_ui')
+
+
+class _StatusUi(object):
+
+    """Gives the user status feedback.
+    The LED and optionally a trigger sound tell the user when the box is
+    ready, listening or thinking.
+    """
+
+    def __init__(self, led_fifo=_LED_FIFO):
+        self.trigger_sound_wave = None
+        if led_fifo and os.path.exists(led_fifo):
+            self.led_fifo = led_fifo
+        else:
+            if led_fifo:
+                logger.warning(
+                    'File %s specified for --led-fifo does not exist.',
```

Comment:
We should change this message since it won't necessarily be true for people using _StatusUi as part of their own code.

Maybe "File %s specified for led_fifo does not exist." or "%s does not exist, disabling status LED."